### PR TITLE
fix(a380x): Add RTPI + ISIS to a380x

### PR DESCRIPTION
**Pull request by flogross89**
Removes dependencies from a32nx, enabling to install and use the A380X without having the A32NX installed.
Additionally, cleans up the fonts for the a32nx and a380x.

## Summary of Changes

- Move all instruments which were loaded from the A32NX to the A380X and modify path in panel.cfg
- Do this for RTPI and ISIS

## Additional context

Discord username (if different from GitHub):

## Testing instructions

**A32NX:**
1. Verify that fonts in all instruments, including EFB and Clock, loaded correctly

**A380X:** 
1. Delete a32nx from community folder. 
2. Load A380X in sim, 
3. Verify that ISIS works and rudder trim indicator is displayed (can't be modified yet though)
4. Verify that fonts in all instruments, including EFB and Clock, loaded correctly
5. Also, please perform a Tier 2 test afterwards. 

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Comment by Popespice** [comment]
Quality Assurance Tester Report

Discord Username  : popespice
Object of testing : #8868
Aircraft          : A380X
Tier of Testing   : 2
Date              : 02/09/2024 

Testing Process: 
1. Log into sim as normal
2. Select any airport
3. Power plane on as normal
4. After complete startup, observe ISIS is working as expected
5. Take off, perform traffic pattern, engage isis mid-flight, change to LS/D mode, change units to hPa and Hg
6. Obersved results: Working as expected

Testing Results: 
Passed

Conclusions: 
Working as expected

Media:

**Review by tracernz — CHANGES_REQUESTED** [review]
Nothing in `fbw-a380x/src/base` should clobber A32NX files in the VFS.

**Comment by tracernz** [comment]
A lot of this cruft can be ignored https://github.com/flybywiresim/aircraft/pull/8873

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Fonts/AirbusRTPI.ttf
[the remark this answers is not in the document]
Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Fonts/AirbusRTPI.ttf:1]: should be resolved after the latest batch of changes

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Pages/VCockpit/Instruments/LegacyA380X/FCU/legacy/A32NX_BaseAirliners.js [hunk 1]
```diff
@@ -0,0 +1,1260 @@
...
+    connectedCallback() {
+        super.connectedCallback();
+        this.preserveAspectRatio("Mainframe");
+        this.addEventAlias("FMS_Upper_INC", "NavigationSmallInc");
+        this.addEventAlias("FMS_Upper_DEC", "NavigationSmallDec");
+        this.addEventAlias("FMS_Lower_INC", "NavigationLargeInc");
+        this.addEventAlias("FMS_Lower_DEC", "NavigationLargeDec");
+        this.addEventAlias("FMS_Upper_PUSH", "NavigationPush");
+        this.radioNav = new RadioNav();
+        this.radioNav.init(NavMode.FOUR_SLOTS);
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Pages/VCockpit/Instruments/LegacyA380X/FCU/legacy/A32NX_BaseAirliners.js:16]:
```suggestion
```

Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Pages/VCockpit/Instruments/LegacyA380X/FCU/legacy/A32NX_BaseAirliners.js]: yeah, also stumbled over that :) thanks!

**Review by tracernz — APPROVED** [review]

**Comment by tylerclt** [comment]
Quality Assurance Trainee Report

Discord Username: tylerb780#3594 
Object of testing : #8868  
Aircraft: A32NX
Tier of Testing: 2
Date: 03/09/2024 

Testing Process: 
1. Loaded into CYYZ for a flight to KBOS
2. Powered on Aircraft and performed a normal preflight 
3. Verified all fonts on the instruments, EFB, and clock worked correctly
6. Departed 
7. Climbed to Cruise
8. Started descent on the JFUND2 STAR
9. Started the approach
10. Landed and parked. 

Testing Results: 
Passed 

Conclusions: 
Works as intended!

**Pull request merged**